### DISCUSSION TITLE
Rails Help with planetscale. Error creating records in DB

### DISCUSSION THREAD

**Discussion by stevesizer** [planetscale/discussion#344] [Q&A]
Hey guys, 

New to planetscale. I have created a user table and can see that on my main branch. When I use the rails console to create a new user i get the following error.

/Users/stevesizer/.rbenv/versions/3.1.1/lib/ruby/gems/3.1.0/gems/mysql2-0.5.4/lib/mysql2/client.rb:148:in `_query': Mysql2::Error: target: testdb.-.primary: vttablet: rpc error: code = NotFound desc = Table 'testdb.application_records' doesn't exist (errno 1146) (sqlstate 42S02) (CallerID: planetscale-admin): Sql: "show full columns from application_records", BindVars: {REDACTED} (ActiveRecord::StatementInvalid) 

Any help would be appreciated.

**Comment by jgreet** [comment]
Hey @stevesizer 

I'm not sure this is a PlanetScale specific problem. Something in your user creation is looking for a table named `application_records` which doesn't exist in your dev branch.

The name stands out because `ApplicationRecord` is the abstract class Active Record models inherit from by default:
https://guides.rubyonrails.org/active_record_basics.html#creating-active-record-models

Maybe `ApplicationRecord` isn't an abstract class? You might need to add a call to [primary_abstract_class](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Inheritance/ClassMethods.html#method-i-primary_abstract_class) in `app/models/application_record.rb`.

See also the current template for application_record.rb:
https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/app/models/application_record.rb.tt
and https://stackoverflow.com/questions/37771127/rails-5-pgundefinedtable-error-relation-application-records-does-not-exi
